config/manager/manager.yaml

```diff
@@ -1,33 +1,26 @@
-apiVersion: v1
-kind: Namespace
-metadata:
-  labels:
-    control-plane: controller-manager
-  name: system
----
```

Comment:
I see why this is a problem, but I think the kubebuilder uses this namespace intentionally with the expectation that you will add a `kustomization` which adds a namespace prefix. e.g.

```
# Adds namespace to all resources.
namespace: e2e

# Value of this field is prepended to the
# names of all resources, e.g. a deployment named
# "wordpress" becomes "alices-wordpress".
# Note that it should also match with the prefix (text before '-') of the namespace
# field above.
namePrefix: e2e-

# Labels to add to all resources and selectors.
#commonLabels:

```

See:
 * https://github.com/kubernetes-sigs/kustomize/blob/master/examples/wordpress/kustomization.yaml

Reply:
So consider just updating the `kustomization.yaml` flie instead.

Reply:
i found it puzzling why the default value was not `default`